sqlglot/generator.py

```diff
@@ -50,9 +50,6 @@ class Generator:
             The default is on the smaller end because the length only represents a segment and not the true
             line length.
             Default: 80
-        annotations: Whether or not to show annotations in the SQL when `pretty` is True.
```

Comment:
we should have an option to strip comments

Reply:
comments: True by default, you can pass in comments: False to strip comments

Reply:
Alternatively, I was thinking:

- Always strip if we're not in pretty mode, or if the comment doesn't contain any newlines. The reason is that:
  - For the former, we want to use /**/ comments and it'd look nice if we ensured that we'll always have 1 leading / trailing space.

  - For the latter, I think preserving leading / trailing space would be unnecessary because the comment appears in a single line and so it's not like we're discarding away important information (?).

- Otherwise, strip comment conditionally, like you suggested.

Does that make sense?

Reply:
i don't mean strip(), i mean remove the comments

Reply:
so False means remove comments entirely

Reply:
Ah ok I'll get it back in then.